qa/pull-tester/rpc-tests.sh

```diff
@@ -93,6 +93,10 @@ testScripts=(
   'sbh_rpc_cmds.py'
   'sc_cert_ceasing.py'
   'sc_getscinfo.py'
+  'quality_mempool.py'
```

Comment:
In order to conform to other test naming, pls rename them adding the 'sc_' prefix